buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -110,10 +140,10 @@ class ClusterFormationTasks {
      *
      * @return a task which starts the node.
      */
-    static Task configureNode(Project project, Task task, NodeInfo node) {
+    static Task configureNode(Project project, Task task, Object dependsOn, NodeInfo node, Configuration configuration) {
 
         // tasks are chained so their execution order is maintained
-        Task setup = project.tasks.create(name: taskName(task, node, 'clean'), type: Delete, dependsOn: task.dependsOn.collect()) {
+        Task setup = project.tasks.create(name: taskName(task, node, 'clean'), type: Delete, dependsOn: dependsOn) {
```

Comment:
My point is since you added the overall clean task (which wipes all of build/cluster), there is no reason to have this per node clean task which wipes build/cluster/nodeN. The only thing here not handled by the global clean is creation of the node dir, but then having a Delete task doesn't seem right just to do a doLast.